scipy/stats/_continuous_distns.py

```diff
@@ -8703,6 +8703,15 @@ def _logsf(self, x, a, b):
             logsf[i] = np.log1p(-np.exp(self._logcdf(x[i], a[i], b[i])))
         return logsf
 
+    def _entropy(self, a, b):
+        A = sc.ndtr(a)
+        B = sc.ndtr(b)
+        Z = B - A
+        C = np.log(np.sqrt(2 * np.pi * np.e) * Z)
```

Comment:
```suggestion
        C = np.log(np.sqrt(2 * np.pi * np.e)) + np.log(Z)
```

Reply:
Is this for readability? It does look slightly more readable with this change, but it doesn't seem right not to keep Z with the other term inside one instance of log.

Reply:
Theoretically, it is possible that $\ln(\sqrt{2\pi e}Z)=\ln(\sqrt{2\pi e})+\ln Z$ overflows. Taking the logarithm apart solves that:

```
np.log(np.sqrt(2 * np.pi * np.e) * np.finfo(np.float64).max) # RuntimeWarning: overflow encountered in double_scalars
np.log(np.sqrt(2 * np.pi * np.e)) + np.log(np.finfo(np.float64).max) # 711.2016514265887
```

Of course this might seem like an artificial case but taking logarithms apart is good practice anyway and this is a low hanging fruit.

Reply:
The flip side to that is that taking the logarithm apart puts us at risk of underflow. I'm also not too sure about taking logarithms apart being best practice. It's a very small difference, but calling ``np.log`` twice instead of once is slower. It's not a huge deal, but I'd rather not split the logarithm.